aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -106,6 +108,50 @@ Tensor isinf(const Tensor &self) {
   });
 }
 
+static inline TensorIterator build_is_infinity_op_iterator(Tensor& result, const Tensor& self) {
+  return at::TensorIteratorConfig()
+    .check_all_same_dtype(false)
+    .set_check_mem_overlap(true)
+    .add_output(result)
+    .add_input(self)
+    .cast_common_dtype_to_outputs(true)
+    .build();
+}
+
+Tensor isposinf(const Tensor &self) {
+  Tensor result = at::ones_like(self, at::kBool, at::MemoryFormat::Preserve);
+  at::native::isposinf_out(result, self);
```

Comment:
For the non `*_out` ops, please call `at::*_out` instead of `at::native::*_out` inside the function, this should fix the xla test failures. Thanks!

Reply:
Thanks for the kind advice, the XLA CI passed after changing `at::native::*_out` to `at::*_out`.